### DISCUSSION TITLE
Image captions only visible for aligned images

### DISCUSSION THREAD

**Discussion by PaulBlenkhorn** [enzedonline/quill-blot-formatter2#17] [Ideas]
Is there any reason why image captions should only be visible for aligned images.  I would find it really useful to have them visible for some images that are inline and not aligned.
("For aligned images (assuming your image blot is inline), the image title is copied into the data-title attribute of the wrapping <span> tag.")

**Comment by enzedonline** [comment]
The caption lives in the `::after` psuedo-element of the span wrapper because `<img>` tags are [replaced elements](https://developer.mozilla.org/en-US/docs/Web/CSS/Replaced_element) - css that modifies the content of replaced elements is unsupported since the browser replaces the content of the tag on render. There's no other place to render a caption on the image element directly.

You could create a custom 'unaligned' format that adds the wrapper without any alignment. You'd probably need to tinker with the css for that format to get it right.
